pkg/operator/context/apis.go

```diff
@@ -25,20 +25,16 @@ import (
 	"github.com/cortexlabs/cortex/pkg/lib/errors"
 	"github.com/cortexlabs/cortex/pkg/lib/hash"
 	"github.com/cortexlabs/cortex/pkg/lib/pointer"
-	"github.com/cortexlabs/cortex/pkg/lib/sets/strset"
 	s "github.com/cortexlabs/cortex/pkg/lib/strings"
 	"github.com/cortexlabs/cortex/pkg/operator/api/context"
 	"github.com/cortexlabs/cortex/pkg/operator/api/resource"
 	"github.com/cortexlabs/cortex/pkg/operator/api/userconfig"
 	"github.com/cortexlabs/cortex/pkg/operator/config"
 )
 
-var uploadedRequestHandlers = strset.New()
-
 func getAPIs(config *userconfig.Config,
 	deploymentVersion string,
 	impls map[string][]byte,
```

Comment:
I don't think we need `impls` or `uploadRequestHandler()` or api.RequestHandlerImplKey any more, right?